From 02346b3c44d4e1de55f07612014281fb76e56de8 Mon Sep 17 00:00:00 2001
From: Khaled Hosny <khaled@aliftype.com>
Date: Thu, 2 Feb 2023 14:19:08 +0200
Subject: [PATCH 05/15] doc: fix cairo_dwrite_font_face_set_rendering_params
 docs

Gtk-doc wants the header and code parameter names to match.

src/win32/cairo-dwrite-font.cpp:1484: warning: Parameter description for cairo_dwrite_font_face_set_rendering_params::params is not used from source code comment block.
src/win32/cairo-dwrite-font.cpp:1484: warning: Parameter description for cairo_dwrite_font_face_set_rendering_params::param is missing in source code comment block.
---
 src/cairo-dwrite.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cairo-dwrite.h b/src/cairo-dwrite.h
index 630fcf280..b1ff718a0 100644
--- a/src/cairo-dwrite.h
+++ b/src/cairo-dwrite.h
@@ -51,7 +51,7 @@ cairo_public IDWriteRenderingParams *
 cairo_dwrite_font_face_get_rendering_params (cairo_font_face_t *font_face);
 
 cairo_public void
-cairo_dwrite_font_face_set_rendering_params (cairo_font_face_t *font_face, IDWriteRenderingParams *param);
+cairo_dwrite_font_face_set_rendering_params (cairo_font_face_t *font_face, IDWriteRenderingParams *params);
 
 cairo_public DWRITE_MEASURING_MODE
 cairo_dwrite_font_face_get_measuring_mode (cairo_font_face_t *font_face);
-- 
2.34.1

